### PR TITLE
Phoenix print

### PR DESCRIPTION
**Pull request by monodo**

### CODE REVIEW

File: geocity/apps/django_wfs3/urls.py [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
-    r"permits_point", api_views.SubmissionPointViewSet, "submissions_point"
+    r"submissions_point", api_views.SubmissionPointViewSet, "submissions_point"
 )
 wfs3_router.register(
-    r"permits_line", api_views.SubmissionLineViewSet, "submissions_line"
+    r"submissions_line", api_views.SubmissionLineViewSet, "submissions_line"
 )
 wfs3_router.register(
-    r"permits_poly", api_views.SubmissionPolyViewSet, "submissions_poly"
+    r"submissions_poly", api_views.SubmissionPolyViewSet, "submissions_poly"
```
Comment (rbovard) [inline geocity/apps/django_wfs3/urls.py:14]: `poly` means nothing, why not `area`?

Comment (sephii) [reply geocity/apps/django_wfs3/urls.py:14]: @monodo I have no clue about this, could you answer so i can then merge teh PR?

Comment (monodo) [reply geocity/apps/django_wfs3/urls.py:14]: Good idea, but out of scope of the refactor

Comment (rbovard) [reply geocity/apps/django_wfs3/urls.py:14]: Yes but we might as well take advantage of it since we rename the urls?

Comment (monodo) [reply geocity/apps/django_wfs3/urls.py:14]: `Poly ` is not ideal, but closer to OGC geometry types definition thus, not sure renaming to area is a great idea. Correct name would by `Polygon `or `MultiPolygon`

Comment (monodo) [reply geocity/apps/django_wfs3/urls.py:14]: Anyway, we'll not rename now, but potentially at the end of refactor if time remains

Comment (rbovard) [reply geocity/apps/django_wfs3/urls.py:14]: Yes but in this case you should also rename line to polyline ;)

**Pull request merged**